Lib/zipfile.py

```diff
@@ -1110,6 +1111,14 @@ def seek(self, offset, whence=0):
             if self._decrypter is not None:
                 self._init_decrypter()
 
+        if read_offset > 0 and self._compress_type == ZIP_STORED and self._decrypter == None:
+            # disable CRC checking after first seeking - it would be invalid
+            self._expected_crc = None
+
+            self._fileobj.seek(read_offset, os.SEEK_CUR)
+            self._left -= read_offset
+            self._offset = read_offset = 0
```

Comment:
here and above (when initializing `self._orig_crc`), please don't assign to multiple variables on one line.  use one line for each, otherwise it is very confusing to read and easy to miss.

```
self._offset = 0
read_offset = 0
```

Reply:
> Does the test_zipfile.py unittest explicitly cover this case of seeking within a ZIP_STORED entry?

I don't think so and I fixed encryption fails (I missed this) in last commit, it passes now on my side. Could you please schedule re-test?

> please don't assign to multiple variables on one line

Ok, I'll fix this later, thanks ;)

Reply:
Fixed assigning multiple variables and squashed both commits.